index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style/style.css" />
+    <script
+      src="https://kit.fontawesome.com/14f4680366.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Artex</title>
+  </head>
+  <body>
+    <header>
+      <div class="icon-holder">
+        <img src="./images/Icons/fb.png" alt="facebook logo" />
+        <img src="./images/Icons/twit.png" alt="twitter logo" />
+        <h4>English</h4>
+        <h4>My Page</h4>
+        <h4>Logout</h4>
+      </div>
+      <nav class="mobile-nav">
+        <img
+          id="mobile-menu"
+          src="./images/Icons/Unionburger_icon.png"
+          alt="mobile menu"
+        />
+        <img
+          id="burger-x"
+          src="./images/Icons/burger-X.png"
+          alt="mobile menu"
+        />
+      </nav>
+      <nav class="desktop-nav">
+        <a href="#">
+          <img src="./images/Icons/logo.png" alt="Artex logo" />
+        </a>
+        <ul>
+          <li>
+            <a id="aboutlink" href="./about.html"><span>About</span></a>
+          </li>
+          <li><a href="">Program</a></li>
+          <li><a href="">Join</a></li>
+          <li><a href=""> Sponsor</a></li>
+          <li><a href=""> News</a></li>
+          <li id="campaign">Artex Campaign</li>
+        </ul>
+      </nav>
+    </header>
+    <section class="home-section">
+      <h4>"Hello! Aesthetes"</h4>
+      <div class="h1-div">
+        <h1>Artex GLOBAL ART EXHIBITION 2022</h1>
+      </div>
+      <p class="home-p1">
+        A joyful celebration believing in the value of openness and sharing,
+        creating positive change with people from over 80 countries is taking
+        place in December, in Greece
+      </p>
+      <h2>2022.12.23(FRI) ~ 24(SAT)</h2>
+      <p class="home-p2">@ Jeff Koons: Apollo Hydra, Greece</p>
+    </section>
+    <section class="main-program">
+      <h3>Main Program</h3>
+      <hr class="hr1" size="1" width="7%" color="#fa6a31" />
+      <div class="program-holder"></div>
+      <a id="main-a" href="">SEE THE WHOLE PROGRAM</a>
+      <button>Join Artex Global Exhibition 2022</button>
```

Comment:
Hello good job working on this project the design is neat but there is something you missed out in this section. It would be nice if you added a background image just like it is in the design template for this project.

This is your design: 

![Screen Shot 2022-10-19 at 18 43 19](https://user-images.githubusercontent.com/79506105/196753621-a63d718a-45fd-4e79-9ca5-d2a6d5d1a344.png)


Your design should look like this:
![mid section](https://user-images.githubusercontent.com/79506105/192027958-033c8126-b54c-42a4-8e3b-1b382ac99f55.png)



You can add a back ground image by doing the folowing 
i. Screenshot a small portion of the background image. It doesn't even need to be bigger than your finger.
ii. Apply the background image using CSS
iii. For it to work properly you must make the background image [repeat](https://developer.mozilla.org/en- 
      US/docs/Web/CSS/background-repeat)

you can use this back ground image:

![190868727-bb97ec1a-97e6-4c80-8698-2b3a9080ddc7](https://user-images.githubusercontent.com/79506105/192027488-823e2fb0-44d2-4312-9578-df493b970036.png)